graph-viz/app/models/Edge.java

```diff
@@ -0,0 +1,79 @@
+package models;
+
+import java.util.Objects;
+
+public class Edge {
+
+    // from Point of edge
+    private Point from;
+    // to Point of edge
+    private Point to;
+
+    public Edge(double fromX, double fromY, double toX, double toY) {
```

Comment:
Since we are using Point and not double, minimize the confusion by actually passing Point not double as the parameters